src/main/js/dynamic/loaded/users/UserTable.js

```diff
@@ -83,12 +83,29 @@ class UserTable extends Component {
 
     onSave() {
         const { user } = this.state;
-        this.props.createUser(user);
+        if (this.checkIfPasswordsMatch(user)) {
+            this.props.saveUser(user);
+            return true;
+        }
+        return false;
     }
 
-    onUpdate() {
-        const { user } = this.state;
-        this.props.updateUser(user);
+    checkIfPasswordsMatch(user) {
+        const passwordKey = 'password';
+        const confirmPasswordKey = 'confirmPassword';
+        const confirmPasswordError = 'confirmPasswordError';
+
+        let passwordError = '';
+        let matching = true;
+        if ((user[passwordKey] || user[confirmPasswordKey]) && (user[passwordKey] !== user[confirmPasswordKey])) {
```

Comment:
I think we want to negate the matching by default, otherwise, the user could save a blank password, no?

Reply:
The only time we want to validate the passwords is if they have provided a value for either password field. If neither password field has been provided, we want to skip this because they may be updating some other User field.